Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserModel.h

```diff
@@ -95,18 +96,25 @@ namespace AzToolsFramework
 
             static void SourceIndexesToAssetIds(const QModelIndexList& indexes, AZStd::vector<AZ::Data::AssetId>& assetIds);
             static void SourceIndexesToAssetDatabaseEntries(const QModelIndexList& indexes, AZStd::vector<AssetBrowserEntry*>& entries);
-            
+
+            void HandleAssetCreatedInEditor(const AZStd::string& assetPath, const AZ::Crc32& creatorBusId = AZ::Crc32());
+
+        Q_SIGNALS:
+            void RequestOpenItemForEditing(const QModelIndex& index);
+
         private:
             //Non owning pointer 
             AssetBrowserFilterModel* m_filterModel = nullptr;
             AZStd::shared_ptr<RootAssetBrowserEntry> m_rootEntry;
             bool m_loaded;
             bool m_addingEntry;
             bool m_removingEntry;
-            bool m_isTickBusEnabled = false;
+			bool m_isTickBusEnabled = false;
```

Comment:
nit: spacing. Might want to do a ctrl+k, cltrl+d to run the standard O3DE formatter which will do all the spacing shenanigans for you

Reply:
Oh, man. That formatting is... yeah, it's formatted alright. Looks like there were tabs in front of that bool instead of spaces. Removed! Hopefully that fixes the problem.